specs/destination_test.go

```diff
@@ -109,7 +109,8 @@ var destinationUnmarshalSpecValidateTestCases = []struct {
 	{
 		"required_name",
 		`kind: destination
-spec:`,
+spec:
+  version: v1.1.0`,
```

Comment:
Unused field? Wouldn't this fail because of `DisallowUnknownFields`?

Reply:
there are no unknown fields. version is known.

Reply:
ah sorry I thought of something like the spec version for some reason